### DISCUSSION TITLE
Modifying ethical costraints of open source models through Axolotl

### DISCUSSION THREAD

**Discussion by Rickybordon0** [axolotl-ai-cloud/axolotl#1923] [Q&A]
Im a pentester and I'm trying to create a tool for my company to implement a locally hosted LLM as a helper for our team.
Is there some way to use axolotl to modify the ethics and legal costraints of the open source models? (even of already instructed models)

**Comment by winglian** [comment]
You should consult a lawyer, but you may want to see if that is within the Use clause of https://ai.meta.com/llama/license/

**Reply by Rickybordon0** [reply]
Yes, that is true. I already asked advice from the legal team. Thank you very much!
In the meanwhile, would it be theoretically possible to do so? If you have any experience with this topic, how complex is it to remove this kind of limitation completly?
